components/meter/meter-radial.js

```diff
@@ -0,0 +1,53 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { heading4Styles } from '../typography/styles.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class MeterRadial extends RtlMixin(LitElement) {
```

Comment:
Note: the `RtlMixin` should only be necessary if the component defines RTL specific styles like...
```
:host([dir="rtl"]) .some-class {
	left: 0;
	right: some-property-value;
}
```
I think the `d2l-meter-linear` might not need this either, yes/no?

Background: the `RTLMixin` simply sprouts `dir="rtl"` on components using the `dir` value from the document element to make these styles work in a scoped/shadow-DOM context.  Previously, Polymer 3 did this for us.  With Polymer 1, this was handled by `host-context`.

Reply:
Yeah, I have to ask Jeff how we should handle rtl for the meters. For now I'll remove them.